packages/react-components/src/permissions/PermissionsProvider.test.tsx

```diff
@@ -0,0 +1,74 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import React from 'react';
```

Comment:
I think you need to add a cond-comp on every line on this file. The stable build CI is getting this error.
![image](https://user-images.githubusercontent.com/79475487/184998469-dd17ff8f-fbaa-43d7-a977-d0c1940eb865.png)
@PorterNan is there a way to easy cond-comp a whole file?

Reply:
done

Reply:
@mgamis-msft, there was, but we removed that feature for some concerns, but maybe we can do this for test only case - because for test there are no entry point, each test.ts is an entry

Reply:
Lemme open a bug for this case, but I don't think this is something we can resolve in this PR

Reply:
Sorry, my mistake. You can remove all cond-comp comments in each line in PermisionProvider.test.tsx. It is okay for PermissionProvider to be part of stable build but just wont be exposed. Also doing this resolves this stable build 
![image](https://user-images.githubusercontent.com/79475487/185248572-b3777c1d-1707-4f95-bbc9-8d0e17fbb15f.png)